kernel/interrupt.c

```diff
@@ -87,6 +87,22 @@ setup_irq_stack (void)
   );
 }
 
+void
+setup_abt_stack (void)
+{
+  asm volatile (
+    "mrs  r0, cpsr \n"
+    "bic  r0, #0x1f \n" // Clear mode bits
+    "orr  r0, #0x17 \n" // Select ABT mode
```

Comment:
For information on the different operating modes see
https://www.keil.com/pack/doc/CMSIS/Core_A/html/group__CMSIS__CPSR__M.html